docs/faq.md

```diff
@@ -109,6 +120,29 @@ labels = query_human(ranks, pool)
 active_dataset.label(ranks, labels)
 ```
 
+## Theory FAQ
+
+Bayesian active learning is a relatively small field will a lot of unknowns. This sections aims at presenting some of our findings so that newcomers can quickly learn.
+
+Don't forget to look at our [literature review](../literature/index.md) for a good introduction to the field.
+
+### Should you use early stopping?
+
+From our experiments, **early stopping hurts the process**. The training dataset is so small that the model overfits very quickly and hence early stopping triggers too early. We also know from [Atighehchian et al.](https://arxiv.org/abs/2006.09916) that **underfitting hurts the process more than overfitting**. 
+
+### Which optimizer works best?
+
+We find that **SGD works well in active learning**. More complex optimizers such as Adam hurt the process. This is mostly the case in the beginning of the process where the model overfits quickly because the training set is small.
```

Comment:
In the Intent classification, I use ADAM and since we reinitialize adam at every AL step it works really well. Let's add a citation to the `label efficiency` paper for the part that we indicate adam hurts since they mention that in their paper but also indicate that based on our experiments we could make AL work for distilled Bert with ADAM.